components/brave_news/browser/urls.cc

```diff
@@ -7,20 +7,12 @@
 
 #include <string>
 
-#include "base/command_line.h"
-#include "brave/components/brave_news/common/switches.h"
+#include "brave/brave_domains/service_domains.h"
 
 namespace brave_news {
 
 std::string GetHostname() {
-  std::string from_switch =
-      base::CommandLine::ForCurrentProcess()->GetSwitchValueASCII(
-          switches::kBraveNewsHost);
-  if (from_switch.empty()) {
-    return "brave-today-cdn.brave.com";
-  } else {
-    return from_switch;
-  }
+  return brave_domains::GetServicesDomain("brave-today-cdn");
```

Comment:
Do `{prod,dev,staging}.brave-today-cdn.com` work? They're different to the current URLs.

You'll probably need to update this URL in a few tests too

Reply:
The tests call `brave_news::GetHostName()` so they should be fine, right?
And your example value is incorrect, but the dev, prod, and staging version of the urls work. Prod is `brave-today-cdn.brave.com`. Staging and Dev change the ending only.

Reply:
Ah I misunderstood what was going on!